packages/cras.rb

```diff
@@ -0,0 +1,64 @@
+require 'package'
+
+class Cras < Package
+  description 'ChromeOS Audio Server'
+  homepage 'https://www.chromium.org/chromium-os/chromiumos-design-docs/cras-chromeos-audio-server'
+  version 'stabilize-12249.B'
+  source_url 'https://chromium.googlesource.com/chromiumos/third_party/adhd/+/refs/heads/stabilize-12249.B/cras/README?format=TEXT'
+  source_sha256 'b65b959f4ad842a219a637413b8702676378a0b5d18b5e413c78c211b3fb133c'
+
+  depends_on 'alsa_lib'
+  depends_on 'ladspa'
+  depends_on 'iniparser'
+  depends_on 'speexdsp'
+  depends_on 'sbc'
+  depends_on 'dbus'
+
+  def self.build
+    system 'git', 'clone', 'https://chromium.googlesource.com/chromiumos/third_party/adhd', '-b', version, '.'
```

Comment:
This should be cloned to a separate directory instead of the current directory since the README file from the source_url is present.

Reply:
`crew` creates a directory called `README.dir` and `chdir`s to that before any of this takes place.

Reply:
Is the directory empty?  If not, the git clone will fail.

Reply:
The directory is empty.